ergoCubSN001/ergocub.xml

```diff
@@ -71,8 +71,8 @@
     <xi:include href="hardware/motorControl/right_leg-eb7-j4_5-mc.xml" disabled_by="disable_right_leg" />
 
     <!-- FT SENSORS -->
-    <xi:include href="hardware/FT/left_arm-eb2-j0_1-strain.xml" disabled_by="disable_left_arm" />
-    <xi:include href="hardware/FT/right_arm-eb1-j0_1-strain.xml" disabled_by="disable_right_arm" />
+ <!--   <xi:include href="hardware/FT/left_arm-eb2-j0_1-strain.xml" disabled_by="disable_left_arm" />
+    <xi:include href="hardware/FT/right_arm-eb1-j0_1-strain.xml" disabled_by="disable_right_arm" /> -->
```

Comment:
Do we want to remove these lines?

Reply:
I prefer not. That is a just a temporary hack since both arm FTs are broken at the moment

Reply:
Gotcha!